.ci/pytorch/test.sh

```diff
@@ -409,7 +409,10 @@ test_inductor_aoti() {
     python3 tools/amd_build/build_amd.py
   fi
   BUILD_AOT_INDUCTOR_TEST=1 python setup.py develop
-  CPP_TESTS_DIR="${BUILD_BIN_DIR}" LD_LIBRARY_PATH="${TORCH_LIB_DIR}" python test/run_test.py --cpp --verbose -i cpp/test_aoti_abi_check cpp/test_aoti_inference
+  # Uncomment next line and delete two more  when https://github.com/pytorch/pytorch/issues/153422 is resolved
+  # CPP_TESTS_DIR="${BUILD_BIN_DIR}" LD_LIBRARY_PATH="${TORCH_LIB_DIR}" python test/run_test.py --cpp --verbose -i cpp/test_aoti_abi_check cpp/test_aoti_inference
+  build/bin/test_aoti_abi_check
+  LD_LIBRARY_PATH=/opt/conda/envs/py_3.10/lib/:$LD_LIBRARY_PATH gdb bin/test_aoti_inference --gtest_filter=AotInductorTest.BasicTestCuda
```

Comment:
`gdb`?

Reply:
I love the smell of gdb in the morning... 
Copy-paste mistake :P